meteor/client/ui/Account/NotLoggedIn/SignupPage.tsx

```diff
@@ -93,11 +93,6 @@ export const SignupPage = translateWithTracker((props: ISignupPageProps) => {
 				email: this.state.email,
 				password: this.state.password,
 				profile: { name: this.state.name },
-				createOrganization: {
```

Comment:
Why remove this? Seems unrelated to this PR and also breaks the functionality in meteor\server\api\user.ts:109

Reply:
I did not see the usage of this property, I think its presence got hidden by uses of a function with the same name in the same file. And this property was not defined in the interface used here, so was being flagged as a type error.

Instead I've added this property to the interface, so it is no longer a type error